### PR TITLE
CV64: Change all mentions of "settings" to "options" and fix a broken link

### PR DESCRIPTION
**Pull request by LiquidCat64**
## What is this fixing or adding?
At @ReverM's request, changes all the mentions of player "settings" that I could find to player "options". Also fixes a link to the options page that is currently broken on the English setup page and changes a couple of wording oddities I happened to notice.

## How was this tested?
Ran the webhost and made sure all the links were working.

### CODE REVIEW

**Review by ReverM — APPROVED** [review]
LGTM
Cannot fully review on the changes in init.py since I'm not a native speaker.

**Comment by LiquidCat64** [comment]
The only thing I changed in init.py was specifying in the game description that it's the Castlevania franchise.

**Review by nicholassaylor — APPROVED** [review]
Looked through all of the changes, links look good, "settings" -> "options" changes look good, `init.py` changes look good

File: worlds/cv64/docs/en_Castlevania 64.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ Enabling Carrie Logic will also expect the following:
 
 - Orb-sniping dogs through the front gates in Villa
 
-Library Skip is **NOT** logically expected on any setting. The basement hallway crack will always logically expect two Nitros
+Library Skip is **NOT** logically expected with any option. The basement arena crack will always logically expect two Nitros
```
Comment (Exempt-Medic) [inline worlds/cv64/docs/en_Castlevania 64.md:119]: The "with any option" here and in `options.py` are tripping me up a bit. I feel like "by any options" and "under any options" are both a bit better. Alternatively, you could just say "X is/are **NEVER** considered in logic"

Comment (LiquidCat64) [reply worlds/cv64/docs/en_Castlevania 64.md]: Changed it to your first suggestion.

**Review by Exempt-Medic — APPROVED** [review]

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**